### PR TITLE
Rebuild all node base images on stretch

### PR DESCRIPTION
**Pull request by gpcastro**
We need to save overall disk space. I'm moving all our node images to be built on top of latest stretch, so they can share its bits.

### CODE REVIEW

File: build/build-runtimeimages-bases.sh [hunk 1]
```diff
@@ -49,18 +49,6 @@ fi
 
 labels="--label com.microsoft.oryx.git-commit=$GIT_COMMIT --label com.microsoft.oryx.build-number=$BUILD_NUMBER"
 
-generateDockerFiles=$(find $runtimeImagesSourceDir -type f -name "generateDockerfiles.sh")
```
Comment (kichalla) [inline build/build-runtimeimages-bases.sh]: I am not clear why this change is made

Comment (gpcastro) [reply build/build-runtimeimages-bases.sh]: It's actually pretty clear... I screwed up :)
